### PR TITLE
fix(TBD-11174): Class not found Log4jFactory

### PR DESCRIPTION
**Pull request by sponomarova**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.hdpx/resources/template/hdpx/hdp3xxTemplate.json [hunk 1]
```diff
@@ -331,41 +331,20 @@
...
-        },
         {
           "groupId": "ch.qos.logback",
           "artifactId": "*",
           "classifier": "",
           "extension": "",
           "description": "remove any logback implementation."
         },
-        {
-          "groupId": "org.apache.logging.log4j",
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.hdpx/resources/template/hdpx/hdp3xxTemplate.json:363]: Same a previous comment. Is it useful to put back log4j2 also ?

File: main/plugins/org.talend.hadoop.distribution.hdpx/resources/template/hdpx/hdp3xxTemplate.json [hunk 2]
```diff
@@ -331,41 +331,20 @@
           "extension": "",
           "description": "remove any slf4j implementation"
         },
-        {
-          "groupId": "log4j",
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.hdpx/resources/template/hdpx/hdp3xxTemplate.json:335]: As I understand missing class org.apache.commons.logging.Logfactory is from commons-logging. Is it useful to put back log4j also ?

Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution.hdpx/resources/template/hdpx/hdp3xxTemplate.json:335]: Modified.

**Review by lbourgeois — APPROVED** [review]
LGTM

**Review by amedvedenko — APPROVED** [review]

**Pull request merged**